modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase

```diff
@@ -48,7 +48,39 @@ definition {
 
 	@description = "LPS-164617. Assert that it's possible to make changes in the FDS "
 	@priority = "4"
-	test CanBeEditted {
+	test CanBeEdited {
+		task ("And Given A custom view is created and selected") {
+			FDSCustomView.createNewView(key_nameView = "New View");
+		}
+
+		task ("When Make some changes to dataset display") {
+			FDSCustomView.selectColumn(key_itemName = "Description");
+
+			FDSFilters.editFilters(
+				key_disableFilters = "Blue",
+				key_enableFilters = "Red",
+				key_filter = "Color");
+		}
+
+		task ("And Click Save button") {
+			FDSCustomView.saveCustomView();
+		}
+
+		task ("Change selection to default view and assert an aspect from default view") {
+			FDSCustomView.viewNoColumn(key_field = "Description");
+
+			FDSCustomView.verifyFilterSummaryLabel(key_filter = "Blue, Green, Yellow");
+		}
+
+		task ("then change selection to editted custom view and assert some aspects of it ") {
+			Click(locator1 = "FrontendDataSet#SELECT_CUSTOM_VIEW");
+
+			FDSCustomView.selectView(key_itemName = "Default View");
+
+			FDSCustomView.viewColumn(key_field = "Description");
+
+			FDSCustomView.verifyFilterSummaryLabel(key_filter = "Green, Yellow, Red");
```

Comment:
This assertion should be "Blue, Green, Yellow". The reason why the test passed is b/c VerifyElementPresent does not fail the test if the element is not present. It will continue with the test and only highlight the step as a warning.